### ISSUE TITLE
Please add Money/Money operator

### ISSUE THREAD

**Issue by oscarfv** [noojee/money.dart#23]
Resulting in a value of type `double`. For calculating ratios.

Thanks.

**Comment by bsutton** [comment]
That's not going to be possible as we already have an Money/num operator  and you can't overload operators in dart.

We could add something like:

```dart
Money one;
Money two;
  double result = one.divideBy(two);
```

However I think the result should actually be a money amount as that would be generally more flexible.

So I'm thinking:

```dart
Money one;
Money two;
  double result = one.divideBy(two).asDouble();
```

Thoughts?

**Comment by bsutton** [comment]
Also can I ask why you are looking to convert to a double?

Generally its considered a bad idea to use a double to hold a monetary amount due to rounding problems.

For transporting and storing money you should use minorUnits.

I'm actually a little wary of adding 'asDouble' as it may encourage users to use it :)

I'm interested to see where your use case fits.

**Comment by oscarfv** [comment]
As you know, a ratio of magnitudes of the same type is dimensionless. If you divide hours by hours (or meters by meters, or carrots by carrots...), you don't get hours, meters or carrots, you simply get a bare number that says how many times A is greater than B.

Too bad that Dart does not support operator overloading. Knowing this, IMHO your suggestion about `.divideBy` looks like the correct solution, supposing that it returns a `double`. (But please keep in mind that I'm a Dart beginner and the value of my opinions are greatly reduced by that fact.)

**Comment by bsutton** [comment]
the method is dividedBy.

good point about the scalers.

**Comment by bsutton** [comment]
released in: 2.0.1-nullsafety.6
